### PR TITLE
eu/migrate templates

### PR DESCRIPTION
**Pull request by Bound3R**
### Description
This PR in a migracion the theme of cypress to eucalyptus 

### Testing
- [x] templates
- [x] extra templates
- [x] assets

### Reviewers
@felipemontoya @jfavellar90 @diegomillan

### CODE REVIEW

**Comment by Bound3R** [comment]
@felipemontoya :eyes:

File: templates/index.html [hunk 1]
```diff
@@ -113,4 +74,4 @@
 <script type="text/javascript">
   $(window).load(function() {$('#signup_action').trigger("click");});
 </script>
-% endif
+% endif
```
Comment (felipemontoya) [inline templates/index.html]: end of line

File: templates/index.html [hunk 2]
```diff
@@ -1,101 +1,62 @@
...
-        <a href="#video-modal" class="media" rel="leanModal">
-          <div class="hero">
-            <div class="play-intro"></div>
+<main id="main" aria-label="Content" tabindex="-1">
+  <section class="home">
+    <header>
+      <div class="outer-wrapper">
+        <div class="title">
+          <div class="heading-group">
+            ## Translators: 'Open edX' is a brand, please keep this untranslated. See http://openedx.org for more information.
```
Comment (felipemontoya) [inline templates/index.html]: we could remove this comment and the next since we als removed the open edx brand

**Review by felipemontoya — CHANGES_REQUESTED** [review]
Some minor changes and we are ready for this. For the sake of order we can also squash into meaningful commits

**Review by felipemontoya — APPROVED** [review]

**Pull request merged**